Source/ChatMgr.cpp

```diff
@@ -319,7 +319,8 @@ namespace {
 
         wchar_t* time_buffer = 0;
         if (localtime.wYear == 0) {
-            time_buffer = Timestamp_seconds ? L"[lbracket]--:--:--[rbracket]" : L"[lbracket]--:--[rbracket]";
+            time_buffer = Timestamp_seconds ? static_cast<wchar_t[]>(L"[lbracket]--:--:--[rbracket]")
```

Comment:
Instead of having `static_cast` here, you can simply (and should) declare `time_buffer` as a `const wchar_t*`.

Reply:
Here the same as my comment above